corelight_md.md

```diff
@@ -1,52 +1,57 @@
-# Collect Corelight sensor logs
+# Collect Corelight Sensor logs
 
-This document describes how you can collect Corelight sensor logs by configuring the Corelight sensor and a Chronicle forwarder. This document also lists the supported log types and supported Corelight versions.
+This document describes how you can collect Corelight sensor logs by configuring the Corelight sensor and a Chronicle forwarder. This document also lists the supported log types and Corelight version.
```

Comment:
This parser supports all Corelight sensors versions from v27.4 and earlier.

Reply:
Updated as per your suggestion.